### PR TITLE
openssl 1.0.2q

### PR DESCRIPTION
**Pull request by alex**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by alex** [comment]
From sampling the errors here, it looks like they're all disk space related?

File: Formula/openssl.rb [hunk 1]
```diff
@@ -4,11 +4,11 @@
 class Openssl < Formula
   desc "SSL/TLS cryptography library"
   homepage "https://openssl.org/"
-  url "https://www.openssl.org/source/openssl-1.0.2p.tar.gz"
-  mirror "https://dl.bintray.com/homebrew/mirror/openssl--1.0.2p.tar.gz"
-  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.0.2p.tar.gz"
-  mirror "http://artfiles.org/openssl.org/source/openssl-1.0.2p.tar.gz"
-  sha256 "50a98e07b1a89eb8f6a99477f262df71c6fa7bef77df4dc83025a2845c827d00"
+  url "https://www.openssl.org/source/openssl-1.0.2q.tar.gz"
+  mirror "https://dl.bintray.com/homebrew/mirror/openssl--1.0.2q.tar.gz"
```
Comment (dskecse) [inline Formula/openssl.rb]: @alex it seems like this 404s

File: Formula/openssl.rb [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
+  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.0.2q.tar.gz"
+  mirror "http://artfiles.org/openssl.org/source/openssl-1.0.2q.tar.gz"
```
Comment (dskecse) [inline Formula/openssl.rb]: @alex could you please replace `http` with `https`?

**Review by dskecse — CHANGES_REQUESTED** [review]
would you mind changing a few things? thank you for your effort!

File: Formula/openssl.rb (hunk 1 above)
Comment (alex) [reply Formula/openssl.rb]: I'm not sure by what process that's updated.

File: Formula/openssl.rb [hunk 3]
```diff
@@ -4,11 +4,11 @@
...
   homepage "https://openssl.org/"
-  url "https://www.openssl.org/source/openssl-1.0.2p.tar.gz"
-  mirror "https://dl.bintray.com/homebrew/mirror/openssl--1.0.2p.tar.gz"
-  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.0.2p.tar.gz"
-  mirror "http://artfiles.org/openssl.org/source/openssl-1.0.2p.tar.gz"
-  sha256 "50a98e07b1a89eb8f6a99477f262df71c6fa7bef77df4dc83025a2845c827d00"
+  url "https://www.openssl.org/source/openssl-1.0.2q.tar.gz"
+  mirror "https://dl.bintray.com/homebrew/mirror/openssl--1.0.2q.tar.gz"
+  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.0.2q.tar.gz"
+  mirror "https://artfiles.org/openssl.org/source/openssl-1.0.2q.tar.gz"
```
[the remark this answers is not in the document]
Comment (dskecse) [reply Formula/openssl.rb]: @javian is this really necessary? it seems like there are other `https` links and they have been there for a while now

[the remark this answers is not in the document]
Comment (javian) [reply Formula/openssl.rb]: There just needs to be one as far as I can tell, not all of them.

**Comment by javian** [comment]
@BrewTestBot test this please

**Pull request closed without merging**